### ISSUE TITLE
Milestone: Train a simple task from human feedback

### ISSUE THREAD

**Issue by timokau** [timokau/prefq#25]
# Our Goal

Our current focus should be to train some behavior end-to-end using our own feedback. The task can be very simple (Pendulum) and we can use imitation for the training.

The "user story" is the following:

- Start server
- Start imitation_example_client.py 
- The imitation example client starts training the agent, regularly sends queries to the server.
- The labeler responds to queries in their web browser (the query client).
- The imitation example client regularly pulls the results and retrains the reward model (imitation takes care of this if the Querent and the Gatherer are implemented correctly).

We can use this for a simple demo with the Pendulum environment. For this, we might need to adjust the hyper-parameters in our example (compare imitation#771). Training the agent should require no more than 200 queries, so (assuming 2s per feedback) < 10 minutes.

# Related issues and PRs

- [x] The RAM issue is a blocker #22.
- [ ] We should update the hyperparameters in the example #28.

**Comment by patrickab** [comment]
Heres the first fully trained pendulum task using [tuned hyperparameter](https://github.com/HumanCompatibleAI/imitation/pull/771)

after 107 evaluations
[pendulum1.webm](https://github.com/timokau/prefq/assets/82589835/a0076159-2b45-4a10-b7d2-5013d26cee79)

after 146 evaluations
[pendulum2.webm](https://github.com/timokau/prefq/assets/82589835/30bbf0ba-1fd1-418e-a01f-6b03b7141323)

after 167 evaluations
[pendulum3.webm](https://github.com/timokau/prefq/assets/82589835/00a78b3e-48a7-4099-a784-e8a8a8552b95)
